embc-app/ViewModels/PrintReferral.cs

```diff
@@ -98,6 +98,7 @@ public class PrintReferral : Referral
         public string ToDate => ValidDates.To.ToString("MMMM-dd-yyyy");
         public string ToTime => ValidDates.To.ToString("h:mm tt");
         public string PrintDate => DateTime.Today.ToString("MMMM-dd-yyyy");
+        public string TotalAmount => TotalAmount.ToString("C2");
```

Comment:
This won't compile, because it has to be renamed.  I'll change it with a coming PR

Reply:
Thanks. I'll leave it with you then. Please add those extra spaces to the templates (other commit in this PR) and then I'll simply close THIS PR.